### PR TITLE
Reduce console output during build

### PR DESCRIPTION
**Pull request by dmitrykuzmin**
This PR addresses #860.

The build output was polished to contain no excessive output from tests that are designed to throw an error.

Also, multiple model verification warnings were either fixed or suppressed with `MuteLogging` where appropriate.

### CODE REVIEW

**Comment by dmitrykuzmin** [comment]
@armiol PTAL.

File: model/model-verifier/src/test/java/io/spine/model/verify/ModelVerifierTest.java [hunk 1]
```diff
@@ -102,6 +102,7 @@ void setUp() {
     }
 
     @Test
+    @MuteLogging
```
Comment (armiol) [inline model/model-verifier/src/test/java/io/spine/model/verify/ModelVerifierTest.java]: Not sure why logging is muted for this test.

File: server/src/test/java/io/spine/server/SubscriptionServiceTest.java [hunk 2]
```diff
@@ -206,6 +206,7 @@ private void checkSubscribesTo(Class<? extends Message> aClass) {
     }
 
     @Test
+    @MuteLogging
     @DisplayName("not subscribe to a system event")
     void notSubscribeToSystemEvent() {
```
Comment (armiol) [inline server/src/test/java/io/spine/server/SubscriptionServiceTest.java]: This test also does not seem to throw "expected" exceptions, so I don't know why mute.

File: server/src/test/java/io/spine/server/aggregate/model/EventApplierTest.java [hunk 3]
```diff
@@ -121,6 +122,7 @@ void hasOneMessageParameter() {
         }
 
         @Test
+        @MuteLogging
```
Comment (armiol) [inline server/src/test/java/io/spine/server/aggregate/model/EventApplierTest.java]: Same as above.

File: server/src/test/java/io/spine/server/command/model/CommandHandlerMethodTest.java [hunk 4]
```diff
@@ -106,6 +107,7 @@ void passNullToleranceCheck() {
     }
 
     @Nested
+    @MuteLogging
```
Comment (armiol) [inline server/src/test/java/io/spine/server/command/model/CommandHandlerMethodTest.java]: Same thing.

File: server/src/test/java/io/spine/server/command/model/CommandHandlerMethodTest.java [hunk 5]
```diff
@@ -201,6 +203,7 @@ void allowEmptyInProcman() {
...
```
Comment (armiol) [inline server/src/test/java/io/spine/server/command/model/CommandHandlerMethodTest.java]: And once again.

File: testutil-server/src/test/java/io/spine/testing/client/grpc/TestClientTest.java [hunk 6]
```diff
@@ -44,6 +45,7 @@
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
+@MuteLogging
```
Comment (armiol) [inline testutil-server/src/test/java/io/spine/testing/client/grpc/TestClientTest.java]: Not sure again. Is this due to a shutdown? Let's comment on muting in this case.

**Comment by dmitrykuzmin** [comment]
@armiol PTAL. I fixed/clarified the usage of `MuteLogging` across the changeset.

File: server/src/main/java/io/spine/server/SubscriptionService.java [hunk 7]
```diff
@@ -120,6 +121,9 @@ private BoundedContext selectBoundedContext(Subscription subscription) {
     private BoundedContext selectBoundedContext(Target target) {
         TypeUrl type = TypeUrl.parse(target.getType());
         BoundedContext result = typeToContextMap.get(type);
+        if (result == null) {
+            throw newIllegalArgumentException("Trying to subscribe to an unknown type: %s", type);
```
Comment (armiol) [inline server/src/main/java/io/spine/server/SubscriptionService.java]: This method is going to be called from `cancel` via `selectBoundedContext(Subscription subscription)`. So the text is misleading.

Also, I don't think that's a good idea to throw an exception. I'd rather return `Optional.empty()`. And if called from `subscribe` I would throw an exception (and return a respective error via gRPC observer's `onError`). But if called from `cancel` I would emit a warning, as such a use case isn't a show-stopper.

File: server/src/test/java/io/spine/server/command/model/CommandHandlerMethodTest.java [hunk 8]
```diff
@@ -216,6 +216,9 @@ void messageParam() {
         }
 
         @Test
+        @MuteLogging
```
Comment (armiol) [inline server/src/test/java/io/spine/server/command/model/CommandHandlerMethodTest.java]: We rarely use single-line comments.

I would rewrite this one (and the one above) like that:

```java
@MuteLogging  /** Signature mismatch warnings are expected. */
```

File: testutil-server/src/test/java/io/spine/testing/client/grpc/TestClientTest.java [hunk 9]
```diff
@@ -45,7 +45,7 @@
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
-@MuteLogging
+@MuteLogging // Mute technical logs about starting the server, etc.
```
Comment (armiol) [inline testutil-server/src/test/java/io/spine/testing/client/grpc/TestClientTest.java]: Let's not mute any logs on starting the server. They are NOT exceptions.

**Comment by dmitrykuzmin** [comment]
@armiol PTAL again.

**Pull request merged**